### DISCUSSION TITLE
MLB Stadiums

### DISCUSSION THREAD

**Discussion by shimizusUW** [jakobzhao/geog495#76] [Final Project]
Github: https://github.com/shimizusUW/geog495_final
Application: https://shimizusuw.github.io/geog495_final/index.html

This application allows the user to find all the MLB ballparks, along with the team that plays there. For a user who wants to learn more about the stadiums and the teams, this application will be of good use. It allows the user to explore the map freely, while also providing a search bar to easily find a specific location. I am doing this as my project because I am the biggest baseball (unfortunately not the Mariners) fan and thought this would be a great way to showcase MLB. 

![Search](https://user-images.githubusercontent.com/49252953/145763152-68f77b9c-b315-449c-b519-6fe821ff5fc5.PNG)

**Comment by FaisalTF** [comment]
I really wanted to do something sports related too, but couldn't find much geo data out there, venues was definitely a good direction! Trying to constructively think about what could improve about this, I think the list could use some more detail, since respective stadium data is being stored either ways, this could also serve as an informative things with basic info (how old maybe? exact address, and more)

**Comment by meleakewubbie** [comment]
I'm also an MLB fan. I like how you ordered the cities in your side panel alphabetically. It's easy to find a specific city with the way you organized it. One aesthetic improvement would be to change the marker for stadiums to something different like the MLB logo or something MLB related.
